spec/defines/check_mk_agent_mrpe_spec.rb

```diff
@@ -11,9 +11,41 @@
     }
   end
 
-  on_supported_os.each do |os, os_facts|
+  on_supported_os.each do |os, facts|
```

Comment:
The use of `os_facts` is preferred nowadays. It avoids an overlap of variable names which can remove confusion.